MaterialSegmentedControl.md

```diff
@@ -56,8 +71,20 @@ This property is to set the color of the unselected segments when it is disabled
 <br/>
 <br/>
 
+### Property AllowMultiselect
+This property is to set if you want to allow multiselect items.
+<br/>
+<br/>
+
 ### Property ItemsSource
-This property is for displaying list of data.
+This property is for displaying list of data. You should set a List of MaterialSegmentedObject.
```

Comment:
I think that it could be MaterialSegmentedItem instead of MaterialSegmentedObject.